eventuate-local-java-common-util/src/main/java/io/eventuate/local/java/common/util/CommonLeaderSelector.java

```diff
@@ -0,0 +1,5 @@
+package io.eventuate.local.java.common.util;
+
+public interface CommonLeaderSelector {
```

Comment:
Call this LeaderSelector or Eventuate 
Add a start() method

Move to package io.eventuate.coordination.leadership

Move it's own Gradle module: eventuate-coordination-leadership

Reply:
will do